packages/components/src/Banner/Banner.module.css

```diff
@@ -58,12 +80,17 @@
 .closeButton {
   margin: calc(var(--space-smallest) * -1) 0;
   align-self: flex-start;
+  mix-blend-mode: multiply;
```

Comment:
blends the hover state with the status color

Reply:
![image](https://github.com/user-attachments/assets/19596f75-3d06-49d4-922c-f501d07f6fdf)